R/scanpyFunctions.R

```diff
@@ -388,17 +389,12 @@ runScanpyPCA <- function(inSCE,
   }
   
   scanpyObject <- zellkonverter::SCE2AnnData(sce = inSCE, X_name = useAssay)
-  if(use_highly_variable == TRUE){
-    scanpyObject$var['highly_variable'] <- 
-      scanpyObject$uns['scanpy']$hvg$var$highly_variable
-  }
-  
   sc$tl$pca(scanpyObject, 
             svd_solver= algorithm, 
             n_comps = as.integer(nPCs), 
             use_highly_variable = use_highly_variable)
   
-  metadata(inSCE)$scanpy$PCA <- scanpyObject
+  inSCE <- zellkonverter::AnnData2SCE(adata = scanpyObject)
```

Comment:
I'm a bit worried that the AnnData2SCE won't keep track of any other info that is stored in the metadata related to our other tools. Could we do something like:
```
tempSCE <- zellkonverter::AnnData2SCE(adata = scanpyObject)
metadata(inSCE)$scanpy_element1 <- metadata(tempSCE)$scanpy_element1
metadata(inSCE)$scanpy_element2 <- metadata(tempSCE)$scanpy_element2
```

to make sure the original inSCE remains intact? You may have to see what elements in the metadata are related to scanpy and make sure they all get transferred over. 

You may have to